lib/elixir/lib/exception.ex

```diff
@@ -647,6 +649,14 @@ defmodule UndefinedFunctionError do
     " is undefined (function #{fa} is not available)"
   end
 
+  def message(%{reason: :"macro with same mfa", module: module, function: function, arity: arity}) do
+    "function " <> Exception.format_mfa(module, function, arity) <>
+    " is undefined but #{inspect(module)} defines a macro with the same name" <>
+    " and arity. Maybe you meant to call:\n\n" <>
```

Comment:
I think the wording "Maybe you meant to call' confusing. I think we should be more assertive one way or the other. For example: "Be sure to require the module if you intend to invoke this macro" or "Make sure you are invoking the correct function".